hack/deployer/runner/eks.go

```diff
@@ -35,7 +35,7 @@ nodeGroups:
   - name: ng-1
     instanceType: {{.MachineType}}
     desiredCapacity: {{.NodeCount}}
-    ami: static
+    ami: {{.NodeAMI}}
```

Comment:
Maybe I'm missing something but what is the benefit of keeping the `nodeAMI` field if it is always set to static ?
In which situations should we set an other value ?

Reply:
2 reasons: 
* just for correctness
* using `auto` solves #3196 but I'm trying to make it work with `static`

The other option, to be logical, is to delete the value that appears in plan.yml. As it is already in place, I thought that we could keep it.